src/AST/Syntax.dfy

```diff
@@ -250,6 +259,7 @@ module Exprs {
     | Block(stmts: seq<Expr>)
     | Bind(vars: seq<string>, vals: seq<Expr>, body: Expr)
     | If(cond: Expr, thn: Expr, els: Expr) // DISCUSS: Lazy op node?
+    | Unsupported(description: string, children: seq<Expr>)
```

Comment:
My thought was that this would usually be some sort of hand-written explanation (maybe also including some auto-generated text from the original expression). But maybe it should just be a representation of the original expression. I'll consider that.

Reply:
I think we should include the position too, and then we can just use C#'s `ToString`

Reply:
Let's create an issue about this

Reply:
#21